### PR TITLE
GT-1052 Minor adjustments for lessons

### PR DESCRIPTION
**Pull request by frett**

### CODE REVIEW

**Review by gyasistory — APPROVED** [review]

File: ui/shortcuts/src/main/java/org/cru/godtools/shortcuts/GodToolsShortcutManager.kt [hunk 1]
```diff
@@ -238,7 +239,10 @@ class GodToolsShortcutManager @VisibleForTesting internal constructor(
 
         val dynamicShortcuts = withContext(ioDispatcher) {
             Query.select<Tool>()
-                .where(ToolTable.FIELD_ADDED.eq(true))
+                .where(
+                    ToolTable.FIELD_TYPE.`in`(*constants(Tool.Type.ARTICLE, Tool.Type.TRACT))
```
Comment (TheNoodleMoose) [inline ui/shortcuts/src/main/java/org/cru/godtools/shortcuts/GodToolsShortcutManager.kt:243]: What is doing exactly?

Comment (frett) [reply ui/shortcuts/src/main/java/org/cru/godtools/shortcuts/GodToolsShortcutManager.kt:243]: it's updating the SQL query that fetches favorited tools. we only want to include tract and article tool types, and exclude the new lesson tool type

Comment (frett) [reply ui/shortcuts/src/main/java/org/cru/godtools/shortcuts/GodToolsShortcutManager.kt:243]: the actual SQL it generates is: `SELECT * FROM tools WHERE type in ('article', 'tract') AND added = true`

Comment (TheNoodleMoose) [reply ui/shortcuts/src/main/java/org/cru/godtools/shortcuts/GodToolsShortcutManager.kt:243]: Ah cool, thanks!

**Pull request merged**